generator/src/main/java/com/scottlogic/deg/generator/validators/VisualisationConfigValidator.java

```diff
@@ -0,0 +1,60 @@
+package com.scottlogic.deg.generator.validators;
+
+import com.google.inject.Inject;
+import com.scottlogic.deg.generator.outputs.targets.FileOutputTarget;
+import com.scottlogic.deg.generator.outputs.targets.OutputTarget;
+import com.scottlogic.deg.generator.utils.FileUtils;
+import com.scottlogic.deg.generator.visualisation.VisualisationConfigSource;
+
+import java.util.ArrayList;
+
+/**
+ * Class used to determine whether the command line options are valid for visualisation
+ */
+public class VisualisationConfigValidator {
+
+    private final FileUtils fileUtils;
+    private final OutputTarget outputTarget;
+    private final VisualisationConfigSource configSource;
+
+    @Inject
+    public VisualisationConfigValidator(FileUtils fileUtils,
+                                        OutputTarget outputTarget,
+                                        VisualisationConfigSource configSource) {
+        this.configSource = configSource;
+        this.outputTarget = outputTarget;
+        this.fileUtils = fileUtils;
+    }
+
+    /**
+     * @return the result of command line validation. Contains a list of error messages.
+     *         if the list is empty then the validation was successful.
+     */
+    public ValidationResult validateCommandLine() {
```

Comment:
Maybe this would be better if it took config source and output target as inputs. The only real dependency is fileUtils.